### PR TITLE
[Doc] Update GraphDataLoader in our docs

### PR DESCRIPTION
**Pull request by hetong007**
## Description
This is a follow up of https://github.com/dmlc/dgl/pull/2496.
It adds `GraphDataLoader` to our tutorials and user guides en/cn.

I'm not sure if updating the examples is necessary, as users using pre-0.6 versions may experience an API break.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [X] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [X] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [X] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

**Comment by BarclayII** [comment]
We can update the examples.  The release branches also contain the examples for that specific version so everything should be fine.

**Comment by hetong007** [comment]
I've done my change on examples as well. Please kindly review that part, thanks!

File: python/dgl/dataloading/pytorch/__init__.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ class GraphDataLoader:
 
     Parameters
     ----------
-    collate : Function, default is None
+    collate_fn : Function, default is None
```
Comment (BarclayII) [inline python/dgl/dataloading/pytorch/__init__.py:430]: You probably don't need this argument explicitly; `DataLoader` also has an argument called `collate_fn`.

**Review by BarclayII — APPROVED** [review]

File: python/dgl/dataloading/pytorch/__init__.py (hunk 1 above)
Comment (hetong007) [reply python/dgl/dataloading/pytorch/__init__.py:430]: I put it here in order to have an easier control over `self.collate`.

**Pull request merged**